service/models.py

```diff
@@ -134,11 +140,11 @@ def deserialize(self, data: dict) -> None:
                 f"Invalid {type(self).__name__}: failed to deserialize request body\nError message: {error}"
             ) from error
 
-    @classmethod
-    def get_by_id(cls, id):
-        """ Get shopcart by primary key: id """
-        logger.info(f"Get {cls.__name__} by id={id}")
-        return cls.query.get(id)
+    # @classmethod
```

Comment:
given get_by_id has been moved to the Base class, should we remove these lines?

Reply:
yes, i just commented these lines in my local branch. I'll remove them in Shopcart class.

Reply:
Actually, I wrote these test cases in test_models.py just like this sample-accounts lol